R/DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: strawr
 Title: Straw is a fast implementation of reading/dump for .hic files
-Version: 0.0.3
+Version: 0.0.4
```

Comment:
thoughts on bumping this to 0.0.8 to match pybind11 version?

Reply:
Good idea; done.